gtsfm/utils/images.py

```diff
@@ -88,7 +88,7 @@ def vstack_image_list(imgs: List[np.ndarray]) -> Image:
     vstack_img = np.zeros((sum(all_heights), img_w, 3), dtype=np.uint8)
 
     running_h = 0
-    for i, img in enumerate(imgs):
+    for img in imgs:
```

Comment:
why the change here to handle non-standard image height? if we're concerned about different image aspect ratios, e.g. Notre Dame, shouldn't we check both height and width? 

In MVS, we have this hard-coded to assume constant image size, so this is a bigger issue.

Reply:
Hmm, the only thing I changed was removing the iterator `i` which wasn't being used. I also removed `img_h`, which also wasn't being used.

Reply:
Oh I see, nevermind. Thanks